docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,542 @@
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/platform/concepts/console_create_service.png
+      :alt: Aiven Console view for creating a new service
+
+3. Select **Apache Kafka®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
+
+5. Enter ``demo-kafka`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console
+
+Customise the Aiven for Apache Kafka service
+''''''''''''''''''''''''''''''''''''''''''''
+
+After creating the service, you'll be redirected to the service details page. You can now customise the service to enable the needed components in the *Overview* tab:
+
+1. **Kafka REST API (Karapace)** > **Enable**
+
+   .. Note:: 
+    The **Kafka REST API** enables you to manage Apache Kafka via REST APIs and also to view the data in your Apache Kafka® topics.
+
+2. **Advanced configuration** > **Add configuration option** > ``kafka.auto_create_topics_enable``, switch the setting on and then click **Save advanced configuration**
+
+   .. Note:: 
+    The ``kafka.auto_create_topics_enable`` setting allows you to create new Apache Kafka® topics as you configure your Apache Flink® data tables, so that you do not need to create the topics in advance.
+
+Create an Aiven for PostgreSQL® service
+'''''''''''''''''''''''''''''''''''''''''
+
+The :doc:`PostgreSQL </docs/products/postgresql>` database is where you'll store the threshold data for each IoT device. To create the database:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+2. On the *Services* page, click **Create a new service**.
+
+3. Select **PostgreSQL®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `hobbyist` as service plan.
+
+5. Enter ``demo-posgresql`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console
+
+
+Create an Aiven for Apache Flink service
+'''''''''''''''''''''''''''''''''''''''''
+
+You will use :doc:`Apache Flink </docs/products/flink>` to define the streaming data pipelines. To create the service:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+2. On the *Services* page, click **Create a new service**.
+
+3. Select **Apache Flink®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
+
+5. Enter ``demo-flink`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console
+
+Customise the Aiven for Apache Flink service
+''''''''''''''''''''''''''''''''''''''''''''
+
+After creating the service, you'll be redirected to the service details page. You can now customise the service to enable the needed integrations to the Aiven for Apache Kafka and Aiven for PostgreSQL services in the *Overview* tab:
```

Comment:
This is the first time you mention integrations between Aiven services.
Maybe this section is not "Customise the Aiven for Apache Flink service", but "Connect the building blocks"